### PR TITLE
6768 redesign of pricing page

### PR DESCRIPTION
**Pull request by denpira**
https://codio.myjetbrains.com/youtrack/issue/codio-6768

### CODE REVIEW

File: source/javascripts/partials/_pricing.es6 [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+  school: 'http://codio-2227229.hs-sites.com/buy-school',
+  university: 'http://codio-2227229.hs-sites.com/buy-university'
+}
+
+const $currency = $('.currency-block .currency');
+const $typeList = $('.nav .nav-tabs');
+const $types = $typeList.find('li.item');
+const $currencyOption = $('.currency-block .currency-option');
+const $tabs = $('.nav-tabs li');
+const $countNumber = $('.pricing-content-col .count .number');
```
Comment (mloginov) [inline source/javascripts/partials/_pricing.es6]: better use
```
$pricingContentCol = $('.pricing-content-col ');
$countNumber = $pricingContentCol.find('.count .number')
const $amount = $pricingContentCol.find('.amount');
const $range = $pricingContentCol.find('.range');
```

File: source/javascripts/partials/_pricing.es6 [hunk 2]
```diff
@@ -0,0 +1,222 @@
...
+    $('.dropdown .dropdown-menu li').find('a[data-index=' + DEFAULT_START[state.type] + ']').click();
+  });
+}
+
+const toggleChevron = e => {
+  $(e.target).prev('.panel-heading').find("i.indicator").toggleClass('glyphicon-toggle-minus glyphicon-toggle-plus');
+}
+
+const setupFaqBlock = () => {
+  $('#accordion-individual').on('hidden.bs.collapse', toggleChevron);
```
Comment (mloginov) [inline source/javascripts/partials/_pricing.es6]: try like this

```
$('#accordion-individual').on('hidden.bs.collapse shown.bs.collapse', toggleChevron);
```

File: source/pricing.html.erb [hunk 3]
```diff
@@ -1,147 +1,147 @@
...
+        <b>No problem, <%= mail_to 'help@codio.com', 'contact us', class: '' %> for a personal demo</b>
+      </div>
+
+      <div class="payment-cards">
+        <%= image_tag 'assets/pricing/visa.png', class: '' %>
+        <%= image_tag 'assets/pricing/mastercard.png', class: '' %>
+        <%= image_tag 'assets/pricing/paypal.png', class: '' %>
+      </div>
+
+      <!--
```
Comment (mloginov) [inline source/pricing.html.erb]: remove unused code

Comment (denpira) [inline source/pricing.html.erb]: This code is disabled temporarily. His fate is not yet clear...

File: source/javascripts/partials/_pricing.es6 (hunk 1 above)
Comment (denpira) [inline source/javascripts/partials/_pricing.es6]: Done

File: source/javascripts/partials/_pricing.es6 (hunk 2 above)
Comment (denpira) [inline source/javascripts/partials/_pricing.es6]: Done

**Comment by mloginov** [comment]
:+1:

**Pull request merged**